PKGBUILD

```diff
@@ -21,15 +19,13 @@ source=(displaylink-driver-${pkgver}.zip::https://www.synaptics.com/sites/defaul
         DISPLAYLINK-EULA
         udev.sh
         99-displaylink.rules
-	displaylink.service
-        displaylink-sleep.sh)
+        displaylink.run)
 sha256sums=('22c552ead448c80d9e8dd48a842bb511184d07a74180ac76bd89dd144ddda816'
             'b5a1a75b2042cd5efb475b53f1ead5207f706c6eb45f4572d7b226ffcdee3ee9'
             '2f81fea43332a62b2cf1dd47e56ea01caf1e886bcd16c3f82b18bfe148fb21a9'
-            'dc41ae8a2c287fc50fdda65bad8b0ffd76726f7773c25e1b0c5b7de95cecbdb6'
+            SKIP
```

Comment:
I never worked with sha256sums before, I will need to figure out how to update this.

Reply:
Just run run:
`updpkgsums`
`makepkg --printsrcinfo > .SRCINFO `(for generating the .SRCINFO)

Reply:
Awesome, that's easy!